AddedMatlabAnalysisFunctionsAN/Bootstrapping.m

```diff
@@ -0,0 +1,52 @@
+%A script to perform bootstrapping on the data in peaks
+%in order to get a better estimate of the error
+% function [aver, err] = BootStrapping(matfile) % returns a scalar, which is the value of the error across all the peaks in a given matfile, calculated using bootstrapping
+% Niter - number of times to pool from the data set
+% nsample - size of the sample to pool from the data set
+% load outall_1_new.mat;
+% useonly - if want to exclude some mages from the analysis; not really used
+% col - which column to normalize by the DAPI column in the 'peaks' cell array). The
+% numbering refers to the numbering of peaks{ii}(:,col) and needs to be
+% verified before plotting , i.e. which channel corresponds to which
+% exactly column in peaks cell array.
+function [aver, err, alldata]=Bootstrapping(peaks,Niter,nsample,col)
+
+
+%if ~exist('useonly','var')
+ %   useonly=1:length(peaks);
+%end
+
+nlines=zeros(length(peaks),1);
+for ii=1:length(peaks)
+    nlines(ii)=size(peaks{ii},1);
+end
+alllines = sum(nlines);
+alldata=zeros(alllines,1);
+q=1;
+size(peaks{2},2);
+for ii=1:length(peaks)
+    if ~isempty(peaks{ii}) && size(peaks{ii},2)>10 %if ismember(ii,useonly) &&  ~isempty(peaks{ii})
```

Comment:
why are we checking if size > 10, what if there was only 1 or 2 antibodies quantified?